superset/contrib/docker/superset_config.py

```diff
@@ -152,3 +152,26 @@ def mutator(f):
 # This parameter documented as deprecated but frontend still uses it
 # Should match `--timeout` value of gunicorn
 SUPERSET_WEBSERVER_TIMEOUT = 300
+
+if get_env_variable('OAUTH_PROVIDER', False) == 'google':
+    from flask_appbuilder.security.manager import AUTH_OAUTH
+
+    AUTH_TYPE = AUTH_OAUTH
+
+    OAUTH_PROVIDERS = [
+        {'name':'google', 'icon':'fa-google', 'token_key':'access_token',
+            'remote_app': {
+                'consumer_key': get_env_variable('OAUTH_CONSUMER_KEY'),
+                'consumer_secret': get_env_variable('OAUTH_CONSUMER_SECRET'),
+                'base_url':'https://www.googleapis.com/oauth2/v2/',
```

Comment:
please format the code

Reply:
It passes flake8 now (I didn't find a specific flake8 config in the repo)

Reply:
https://github.com/src-d/sourced-ui/pull/292/checks?check_run_id=244877259
sourced-ui uses flake8, pylint and black.

Reply:
flake8 and pylint look fine for this change (although I'm not 100% familiar with pylint).

This file doesn't follow black conventions, so I'd leave it as-is, but if you want, I can introduce a commit before applying black to the existing file, then add my change with black conventions.